FrontEndLib/BitmapManager.h

```diff
@@ -68,6 +68,15 @@ struct CustomTiles
 	UINT wTileStartNo;     //starting number of these custom tiles
 };
 
+struct TileMask
+{
+	TileMask(UINT tile, int xOffset, int yOffset)
+		: tile(tile), xOffset(xOffset), yOffset(yOffset)
+	{ }
+	UINT tile;
```

Comment:
Maybe `tileImage`? Though looking at the rest of the code, TI's are usually referred to as tiles.

Reply:
Agree this should be renamed to be more descriptive.
I'm renaming it to TweeningTileMask to indicate it's used for tiles in motion.